devnet/docker-compose.yml

```diff
@@ -11,8 +12,10 @@ services:
       - 8546
       - 8540
       - 9876
+      - 6060
```

Comment:
what is this port for?

Reply:
This is the gossamer node pproof server port

Reply:
Nice addition 👍